admin-part/src/main/kotlin/Plugin.kt

```diff
@@ -116,6 +118,25 @@ class Plugin(
                 ).toActionResult()
             } ?: ActionResult(StatusCodes.NOT_FOUND, "Active session '$sessionId' not found.")
         }
+        is ExportCoverage -> action.payload.runCatching {
+            val result = doAction(StartNewSession(StartPayload(testType = "UNIT")))
+            (result.data as? StartAgentSession)?.payload?.sessionId?.run {
+                val data = Base64.getDecoder().decode(data)
+                val file = File(System.getProperty("java.io.tmpdir"))
```

Comment:
It would be better if variables have more details
data -> fileData/inputData
file -> execFile/exec